xesmf/frontend.py

```diff
@@ -8,7 +8,7 @@
 import numpy as np
 import sparse as sps
 import xarray as xr
-from shapely import LineString
+from shapely.geometry import LineString
```

Comment:
I did not see any deprecation warning when importing like this, so I guess it is safe enough!